### ISSUE TITLE
 The network doesn't work.

### ISSUE THREAD

**Issue by sonsungwon** [fbelavenuto/arpl#5]
my computer info
CPU : i5-12500T
MB : H670M-ITX
lan: Realtek Gaming 2.5GbE Family Controller, 
       Intel(R) Ethernet Connection (17) I219-V

![네트워크 에러](https://user-images.githubusercontent.com/6913952/177351642-19c43b7c-f862-47a8-945e-009776cdeb0e.png)

**Comment by fbelavenuto** [comment]
Hi, thanks for report. I will research which module is for this card and create an addon if possible.

**Comment by fbelavenuto** [comment]
Please, can you run the command "lspci -nnk" and post the result?

**Comment by fbelavenuto** [comment]
Yes, thanks. I saw that the virtual network is from vmware, "e1000". Detection should occur correctly. I'm testing on VMWare workstation and it's working for me.

**Comment by fbelavenuto** [comment]
Please test with fresh 0.3-alpha1 release

**Comment by fbelavenuto** [comment]
Some success?

**Comment by darkguy2008** [comment]
Hello, maybe related/unrelated but I'm having issues trying to get it to boot with my Atheros AR8161 (alx driver in TCRP). Under ARPL it works, but when it boots to DSM it doesn't. Is there a way to add this module/addon?

Thanks for the great work @fbelavenuto ! :)

**Comment by fbelavenuto** [comment]
Hi, this module already exists! Please try update modules, addons and ARPL, rebuild the loader and test.

**Comment by AuxXxilium** [comment]
> Hello The network doesn't work.
> 
> my computer info CPU : i5-12500T MB : H670M-ITX lan: Realtek Dragon RTL8125BG Intel(R) Ethernet Connection (17) I219-V
> 
> ![네트워크 에러](https://user-images.githubusercontent.com/6913952/177351642-19c43b7c-f862-47a8-945e-009776cdeb0e.png)

hi, i have the same MB and both networkcontroller are working out of the box.
